src/udev/udev-node.c

```diff
@@ -26,6 +26,34 @@
 
 #define UDEV_NODE_HASH_KEY SD_ID128_MAKE(b9,6a,f1,ce,40,31,44,1a,9e,19,ec,8b,ae,f3,e3,2f)
 
+int udev_node_cleanup(void) {
+        _cleanup_closedir_ DIR *dir = NULL;
+
+        /* This MUST be called when no worker exists. Otherwise, it causes an race between mkdir()
```

Comment:
I think the comment should be inverted here: "This must not be called when any workers exist. It would cause a race …"

(Right now the sentence says that it's important to call this… but this is not what the comment was trying to convey.)